docs/cli.md

```diff
@@ -78,6 +78,7 @@ that image and add them to the tar streamed to the container into `/artifacts`.
 | `-e (--env)`               | Environment variables to be passed to the builder eg. `NAME=VALUE,NAME2=VALUE2,...` |
 | `-E (--environment-file)`               | Specify the path to the file with environment |
 | `--force-pull`             | Always pull the builder image, even if it is present locally (defaults to true) |
+| `--run`             | Launch the resulting image in a container, where stdout and stderr from the container are piped to the stdout and stderr of sti to help determine the image's validity (along with any docker commands you which to execute on the container provided), and where if the image is long running, you will have to Ctrl-C to exit both sti and the container (defaults to false) |
```

Comment:
I'd suggest to rewrite it like this:
Launch the resulting image after a successful build. All output from the image is being printed to help determine image's validity. In case of a long running image you will have to Ctrl-C to exit both sti and the running container.